tests/data_context/store/test_ge_cloud_store_backend.py

```diff
@@ -44,6 +46,115 @@ def _closure(resource_type: GeCloudRESTResource) -> GeCloudStoreBackend:
     return _closure
 
 
+@pytest.mark.parametrize(
+    "base_url,organization_id,resource_name,id,expected",
+    [
+        pytest.param(
+            "https://app.test.greatexpectations.io"
+            "de5b9ca6-caf7-43c8-a820-5540ec6df9b2",
+            "my_resource",
+            "my_value",
```

Comment:
Looks like something funky is going on - maybe a missing comma at the end of the base_url param (there are 5 param names but 6 provided)